src/witan/models/dem/ccm/fert/hist_asfr_ws.clj

```diff
@@ -5,9 +5,9 @@
             [witan.models.dem.ccm.schemas :refer :all]))
 
 ;; GET DATA FOR ROOT NODES
-(def root-data-paths {:births-data "resources/test_data/bristol_births_data.csv"
-                      :at-risk-popn "resources/test_data/bristol_denominators.csv"
-                      :hist-births-est "resources/test_data/bristol_hist_births_est.csv"})
+(def root-data-paths {:births-data "test_data/bristol_births_data.csv"
```

Comment:
@acron0 they're only used in this namespace (`hist_asfr_ws.clj`) where we tried to implement the workflow before and in the test namespaces.